lib/initialize.js

```diff
@@ -17,7 +14,7 @@ const Rx = require('rx');
 
 module.exports = function initializeInquirer(pkg) {
 	if(pkg.length == 0) {
```

Comment:
Is default in yargs, in the future, when we're revamping the ui, perhaps. This module is gonna be deleted anyway